packages/compass-connections/src/stores/connections-store.tsx

```diff
@@ -337,7 +335,7 @@ export function useConnections({
   const closeConnection = async (connectionId: string) => {
     debug('closing connection with connectionId', connectionId);
     log.info(
-      mongoLogId(1_001_000_313),
+      mongoLogId(1_001_000_317),
```

Comment:
Same as with another id that was changed, I'm guessing by accident

```suggestion
      mongoLogId(1_001_000_313),
```